README.md

```diff
@@ -46,6 +46,42 @@ bus.bind("order", {
 });
 ```
 
+## Environment Configuration Variables
+
+For more details and up to date information on Kafka specific configs check the [Kafka official documentation](https://kafka.apache.org/documentation/#configuration).
+
+| Name     | Type   | Mandatory | Default | Description |
+| -------- | ------ | ------- | ------- |----------- |
+| **BUS_ADAPTER** | `str`  | false | `kafkaRetry`    | RIPE Bus adapter |
```

Comment:
See how https://raw.githubusercontent.com/ripe-tech/ripe-core/master/README.md?token=AAXILK6FSZSE35VXV2NTHIC74N5FY deals with "enum" (for instance `RIPE_INSTORE_STYLE`)